build.gradle

```diff
@@ -2,6 +2,11 @@ plugins {
     id 'java'
 }
 
+compileJava {
+    sourceCompatibility = 17
+    targetCompatibility = 17
+}
```

Comment:
That's the redundant block, you already specified source and target in lines 26-27

Reply:
fixed